src/main/java/fi/pohina/vinkkilista/utils/TagParser.java

```diff
@@ -0,0 +1,85 @@
+package fi.pohina.vinkkilista.utils;
+
+import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import java.util.List;
+import java.util.Set;
+
+public class TagParser {
+    
+    private static final String DELIMITER = ",";
+    
+    public String tagFromUrl(String url) {
```

Comment:
Is this "tag parsing" behavior? I think this one would fit into the `TagService` better